Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1399,7 +1404,7 @@ open class NavigationMapView: UIView {
                     self.travelAlongRouteLine(to: location.coordinate)
                 }
             default:
-                if self.simulatesLocation, let locationProvider = self.mapView.location.locationProvider {
+                if self.simulatesLocation && self.inActiveNavigation, let locationProvider = self.mapView.location.locationProvider {
```

Comment:
Nit, more _Swifty_ way I guess:
```suggestion
                if self.simulatesLocation, self.inActiveNavigation, let locationProvider = self.mapView.location.locationProvider {
```

Reply:
Thanks for the suggestion! I'll improve this part.